### PR TITLE
Add CEL variable for well known issuers

### PR DESCRIPTION
**Pull request by Celdir**
Prevent mistakes by providing CEL variable for well known OIDC token issuers.

https://github.com/abcxyz/github-token-minter/issues/134

### CODE REVIEW

File: .github/minty.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 'minty.abcxyz.dev/v2'
 
 rule:
   if: |-
-    assertion.iss == 'https://token.actions.githubusercontent.com' &&
+    assertion.iss == issuers.github &&
```
Comment (Celdir) [inline .github/minty.yaml:5]: @drevell is it safe to change this now or do I need to wait for minty to deploy before updating its own config?

Comment (drevell) [reply .github/minty.yaml:5]: Better ask Brad.

Comment (verbanicm) [reply .github/minty.yaml:5]: you will need to redeploy.

**Review by verbanicm — CHANGES_REQUESTED** [review]
change minty config after deployment

**Review by verbanicm — APPROVED** [review]

**Pull request merged**